_issues/rapist.md

```diff
@@ -0,0 +1,21 @@
+---
+layout: post
+title: "Didn't she get a rapist acquitted, and then laugh about the girl who was raped?"
+categories: controversy
+---
+
+**Q:** Didn't Hillary get a rapist acquitted, and then laugh about the girl who was raped?  
+
+**A**: In the second Presidential debate, Donald Trump claimed "Her client she represented
+got him off and she is seen laughing on two occasions laughing at the girl who was raped." This is [completely false](http://www.politico.com/blogs/2016-presidential-debate-fact-check/2016/10/trump-is-wrong-hillary-clinton-did-not-laugh-about-the-rape-of-a-12-year-old-229455#ixzz4MiTn0Cwt). Here are the facts:
```

Comment:
@eyas I'm a little concerned about this video. Although it is very good, it has an audio of the victim's voice attacking Hillary and accusing her of being a hypocrite. It's much later in the tape, and there are two really good quotes from the prosecutor: the one you mentioned and this one:

"She was just doing her job. She was going to present the best defense she could. And she was certainly going to require us to prove his guilt."

But I'm wondering if I should look for an article that has those quotes, rather than link the whole video.

Thoughts?